apps/condo/domains/common/components/UseDeskWidget.tsx

```diff
@@ -4,7 +4,11 @@ import getConfig from 'next/config'
 import get from 'lodash/get'
 
 const { publicRuntimeConfig:{ UseDeskWidgetId } } = getConfig()
-const fieldId = 20560
+const fieldId = {
```

Comment:
What about the more understandable name?
```suggestion
const useDeskFieldsIdsMap = {
```

Reply:
yes you are right, I forgot to change :)